native/src/components/DatePicker.tsx

```diff
@@ -64,9 +64,18 @@ export type DatePickerProps = {
   error?: string
   modalOpen: boolean
   setModalOpen: (open: boolean) => void
+  placeholderDate?: DateTime
```

Comment:
🔧 Since we always pass a placeholder date (and it is best practice to do so), we should make this required.
```suggestion
  placeholderDate: DateTime
```